src/transformState.js

```diff
@@ -5,7 +5,28 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const current of actions) {
+    switch (current.type) {
+      case 'addProperties': Object.assign(state, ...[current.extraData]);
+        break;
```

Comment:
check the code formatting
```suggestion
      case 'addProperties':
        Object.assign(state, ...[current.extraData]);
        break;
```